ktor-http/common/src/io/ktor/http/HttpHeaders.kt

```diff
@@ -4,127 +4,127 @@
 
 package io.ktor.http
 
-@Suppress("unused", "KDocMissingDocumentation", "PublicApiImplicitType", "MayBeConstant")
+@Suppress("unused", "KDocMissingDocumentation", "PublicApiImplicitType")
 public object HttpHeaders {
     // Permanently registered standard HTTP headers
     // The list is taken from http://www.iana.org/assignments/message-headers/message-headers.xml#perm-headers
 
-    public val Accept: String = "Accept"
-    public val AcceptCharset: String = "Accept-Charset"
-    public val AcceptEncoding: String = "Accept-Encoding"
-    public val AcceptLanguage: String = "Accept-Language"
-    public val AcceptRanges: String = "Accept-Ranges"
-    public val Age: String = "Age"
-    public val Allow: String = "Allow"
+    public const val Accept: String = "Accept"
```

Comment:
You can do something like

```kt
    @Deprecated("Use Accept constant instead.", ReplaceWith("Accept"))
    @get:JvmName("getAccept")
    public val _Accept: String = Accept
    public const val Accept: String = "Accept"
```

instead. But I don't think it's very necessary.